### DISCUSSION TITLE
Mongo not running (and not installed)

### DISCUSSION THREAD

**Discussion by pdifeo** [amidaware/tacticalrmm#2023] [Q&A] [answered]
The call at health status page return me this

```
{
  "version": "0.19.3",
  "latest_agent_version": "2.8.0",
  "agent_count": 53,
  "client_count": 5,
  "site_count": 15,
  "disk_usage_percent": 6,
  "mem_usage_percent": 27,
  "days_until_cert_expires": 33,
  "cert_expired": false,
  "redis_ping": true,
  "services_running": {
    "django": true,
    "mesh": true,
    "daphne": true,
    "celery": true,
    "celerybeat": true,
    "redis": true,
    "postgres": true,
    "mongo": false,
    "nats": true,
    "nats-api": true,
    "nginx": true
  }
```
Is Mongo needed? Why wasn't it installed by the installation script?

**Comment by wh1te909** [comment, marked as answer]
it's not needed, but some older installs still haven't migrated yet. it will be removed from the monitoring endpoint eventually

**Reply by pdifeo** [reply]
Thank you, I am preparing a template for Zabbix and wanted to know whether or not I should enable a trigger on Mongo. When the template is ready I will make it available on github.

**Reply by dinger1986** [reply]
I wouldn't monitor for mongo with zabbix.

A template would be great, we can list it in our awesome repo

**Comment by dinger1986** [comment]
Mongo was used for meshcentral until last year but has been replaced by postgresql.

It's still checked for as older installs might have it.

https://github.com/amidaware/tacticalrmm/releases/tag/v0.16.0

**Comment by pdifeo** [comment]
Alpha version of the template is at

https://github.com/pdifeo/tacticalrmm-zabbix
